### ISSUE TITLE
[SUGGESTION] turret targeting all ships and stations, and more bot types

### ISSUE THREAD

**Issue by matejkar** [tsunamayo/Starship-EVO#2786]
One of the things i loved the most about SEVO before steam release was the ability to target any entity with turrets, now it only works for aggressive bots.
 Firstly i would suggest getting back the ability to target anything with turrets, and also to add new bot types, such as civil (just driving around), military (armed ships which drive around until combat occurs within lets say 3-5km) and ally npc's, which will follow the player where he goes and engage the targets he targets.
Im mostly suggesting this because when i got multiplayer working today with a friend, we took 2 ships but i couldnt shoot him with turrets, but before he made a new ship, i gave him my old ship, and then they would target him and shoot.
Of course, i can add stuff like more npc characters, not just ships, more explorable planets, and generally more life in the universe, but also the ability to remove all nps charecters (until the players spawn an aggressive ship)

**Comment by tsunamayo** [comment]
Yes I wanted to add that option, I am not sure on the best way to do this. The last I want is to fire at a random civilian you simply wanted to check...

**Comment by matejkar** [comment]
i understand, maybe add an option on the turret computer, like a list of bot types it can shoot, or make it able to shoot only unmanned ships, and other player's ships.
Forget about the more bot stuff, just maybe an idea for later.
But yes i would be very very grateful, because if im not wrong, the only was for turrets on ships to fire is by spawning aggressive bots with f8, which does get old as they are not usually a threat.
Only bot thing that should be fixed soon is that aggressive bots almost always run away from each other. For example, i spawn 2 large ships as aggressive bots, and they just ignore each other, and hyperspace away.

**Comment by StarrKiss** [comment]
Maybe add a way to target vs check? Say, a simple middle click displays stats, and a hold middle click pulls up an action/pie menu (similar to middle click on blocks) that gives the OPTION to target (as well as to, say, communicate with/hail, and maybe view faction) of the ship.

**Comment by tsunamayo** [comment]
@StarrKiss yes that is one good option.
